src/utils/bugzilla/generateBugzillaUrls.jsx

```diff
@@ -0,0 +1,46 @@
+import { stringify } from 'query-string';
+import fetchJson from '../fetchJson';
+
+const BZ_HOST = 'https://bugzilla.mozilla.org/buglist.cgi';
+const BZ_REST_API = 'https://bugzilla.mozilla.org/rest/bug';
+
```

Comment:
Might be more useful to put these in a `constants.js` file and import them.

Reply:
Because I believe they're module-level values rather than app-level I will keep them here.

Thanks for the review!